backend/src/test/java/com/huellapositiva/util/TestData.java

```diff
@@ -397,9 +402,9 @@ public String registerESALandPublishedProposalObject() throws ParseException {
                 .title("Recogida de ropita")
                 .esal(new ESAL(esal.getName(), new Id(esal.getId())))
                 .location(new Location("SC Tenerife", "La Laguna", "Avenida Trinidad", "12345", "Tenerife"))
-                .startingProposalDate(ProposalDate.createStartingProposalDate("20-01-2021"))
-                .closingProposalDate(ProposalDate.createClosingProposalDate("24-01-2021"))
-                .startingVolunteeringDate(ProposalDate.createStartingVolunteeringDate("25-01-2021"))
+                .startingProposalDate(ProposalDate.createStartingProposalDate("20-11-2021"))
+                .closingProposalDate(ProposalDate.createClosingProposalDate("24-11-2021"))
+                .startingVolunteeringDate(ProposalDate.createStartingVolunteeringDate("25-11-2021"))
```

Comment:
This will just delay the issue to November. It would be a better approach to user relative times instead.
Instant.now().minusDays(1) and so to guarantee that the test will not break depending on which day you run it.

ALREADY FIXED in the develop branch, just bring the changes to your branch

Reply:
Solved